src/main/java/io/appium/java_client/AppiumCommandInfo.java

```diff
@@ -0,0 +1,45 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * See the NOTICE file distributed with this work for additional
+ * information regarding copyright ownership.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.appium.java_client;
+
+import org.openqa.selenium.remote.CommandInfo;
+import org.openqa.selenium.remote.http.HttpMethod;
+
+public class AppiumCommandInfo extends CommandInfo {
+    private final String url;
```

Comment:
@mykola-mokhnach 
No. Take a look at the 
https://github.com/appium/java-client/blob/master/src/main/java/io/appium/java_client/MobileCommand.java#L127

and 

https://github.com/appium/java-client/blob/master/src/main/java/io/appium/java_client/MobileCommand.java#L197

I think that an URL like that can't be created by Java. It throws MalformedException at that case.